src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualNetworkPrivateAccessService.md

```diff
@@ -0,0 +1,64 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmVirtualNetworkPrivateAccessService
+
+## SYNOPSIS
+Lists private access services for location
+
+## SYNTAX
+
+```
+Get-AzureRmVirtualNetworkPrivateAccessService -Location <String>
+```
+
+## DESCRIPTION
+Get-AzureRmVirtualNetworkPrivateAccessService lists private access service values available in the specified location.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmVirtualNetworkPrivateAccessService -Location westus
+
+Name              Id                                                                                                                      Type
+----              --                                                                                                                      ----
+Microsoft.Storage /subscriptions/2c224e7e-3ef5-431d-a57b-e71f4662e3a6/providers/Microsoft.Network/privateAccessServices/Microsoft.Storage Microsoft.Network/privateAccessServices
+```
+
+Gets available private access services in westus region.
+
+## PARAMETERS
+
+### -Location
+The location.
```

Comment:
@EvgenyAgafonchikov please update this description to something a little more meaningful (although I understand it is pretty self-explanatory), such as "The location to retrieve the private access services from."